yarn-project/noir-contracts/src/contracts/non_native_token_contract/src/main.nr

```diff
@@ -102,13 +111,16 @@ contract NonNativeToken {
         let mut initialContext = PrivateFunctionContext::new();
         initialContext.args = initialContext.args.push_array([amount, sender.x, sender.y, recipient]);
 
+        // Gets the set of sender's notes and picks 2 of those.
```

Comment:
The line before: `initialContext.args.push_array` -> should we explain why we do this?